internal/command/import.go

```diff
@@ -248,13 +248,22 @@ func (c *ImportCommand) Run(args []string) int {
 		return 1
 	}
 
+	// Get schemas, if possible, before writing state
+	var schemas *terraform.Schemas
+	if isCloudMode(b) {
+		schemas, diags = c.GetSchemas(newState, nil)
+		if diags.HasErrors() {
+			c.Ui.Warn(fmt.Sprintf(failedToLoadSchemasMessage, err))
```

Comment:
I think these are going to need to error out. The json marshaling code will panic if a nil schema is pass in.

Reply:
Actually I take that back, digging deeper it looks like there were some nil checks added because that's happened before, and it _should_ error out instead, though I don't think purposely sending `nil` is something that was will tested.